bin/ert.in

```diff
@@ -3,20 +3,6 @@ import sys
 import os
 from argparse import ArgumentParser
 
-ERT_ROOT = "${ERT_ROOT}"
-ERT_LD_LIBRARY_PATH = "%s/${CMAKE_INSTALL_LIBDIR}" % ERT_ROOT
-ERT_PYTHONPATH = "%s/${PYTHON_INSTALL_PREFIX}" % ERT_ROOT
-ERT_SHARE_PATH = "%s/share/ert" % ERT_ROOT
```

Comment:
How is the `ERT_SHARE_PATH` handled

Reply:
The ERT_SHARE_PATH is only used for the gui and shell for the location of images and help documentations as far as I can find.

It's now using the previous fall back solution `ert_share_path = os.path.realpath(os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../../../share/ert"))`

This isn't a perfect solution either, especially looking up the tree path is a bit awkward.

Reply:
NB!! The `ERT_SHARE_PATH` variable is (still) also used in the `site-config` file in ert-statoil: https://github.com/Statoil/ert-statoil/blob/master/etc/ERT/site-config.

Reply:
I've reverted the part concerning the ERT_SHARE_PATH, we'll work that out later then.

Reply:
Sounds reasonable..